content/app.md

```diff
@@ -30,6 +30,36 @@ Galaxy records the output of your app's logs in addition to printing operational
 
 You can [filter logs from a specific container](/containers.html#connect-logs) on the app container page.
 
+<h3 id="icon-meanings">Icons</h3>
+
+Icons in your app's dashboard will display the current state of your app.
+
+<img src="/images/green_circle.png" style="margin: 1em 0;"/>
+
+The green full circle indicates that your app's container set is running, and passed its health checks. For an app that should be running, this is the desired state.
+
+<img src="/images/green_circle_outlined.png" style="margin: 1em 0;"/>
+
+The outlined green full circle means that your app's container set status is starting, updating, or stopping. It should leave this state fairly quickly.
+
+<img src="/images/empty_circle_outlined.png" style="margin: 1em 0;"/>
+
+The outlined green empty circle means that your app's container set has experienced an issue while starting.
+
+This can mean:
+- your app has never successfully run all its containers, or has not since the last time it was stopped
+- your app is currently running (possibly unhealthy) containers fewer in number than the amount it is trying to run
+
+Potential causes include a build failure caused by an inability to build your code as written, or a regular crash upon startup. The problem will need to be resolved before your app can successfully run.
+
+<img src="/images/red_circle.png" style="margin: 1em 0;"/>
+
+The red full circle indicates that all the containers your app is trying to run are unhealthy. Your app won't be able to load or run successfully until you resolve the issue.
+
+<img src="/images/gray_circle.png" style="margin: 1em 0;"/>
+
+The gray full circle indicates that all the containers in your app's container set are stopped. If necessary, you can check the Activity Log in your account's right-hand sidebar to determine which user(s) completed this action. You can always restart your containers, at any time.
```

Comment:
This is technically accurate but note that we have two stop operations: "stop a container" and "stop an app".  I think it's easy to think that your sentence is about the former but it's the latter.  Gray circle means "you stopped the entire app, and the process of stopping is done".